src/exp2cxx/expressbuild.cc

```diff
@@ -61,7 +61,7 @@ ComplexCollect::ComplexCollect( Express express )
 
     // Remove all Lists corresponding to entities which are subtypes of other
     // entities.  Such Lists should not exist on their own since their supers
-    // supercede them.  (They were added in the first place to be available
```

Comment:
"Supercede has occurred as a spelling variant of supersede since the 17th century, and it is common in current published writing. It continues, however, to be widely regarded as an error. "

Reply:
@jepler good to know. I'll update the `codespell` folks about this.

Reply:
...though if you do search the stepcode codebase you'll see the rest of the occurrences are `supersede`.  Pushing PR shortly